docs/development/_index.md

```diff
@@ -54,6 +54,109 @@ options can be appended to the main command as arguments.
 make image
 ```
 
+
+### Running the docker image
+
+With docker you can run the Headlamp image. Note, the mount arguments add folders that are referenced in the ~/.kube folders - you may need to add other folders if your config has more.
+
+```bash
+docker run --network="host" -p 127.0.0.1:4466:4466/tcp --mount type=bind,source="/home/rene/.minikube",target=$HOME/.minikube --mount type=bind,source="$HOME/.kube",target=/root/.kube quay.io/kinvolk/headlamp:latest /headlamp/server -html-static-dir /headlamp/frontend -plugins-dir=/headlamp/plugins
+```
+
+If you just made an new docker image you can run it like this:
+
+```bash
+$ DOCKER_IMAGE_VERSION=development make image
+...
+Successfully tagged kinvolk/headlamp:development
+
+$ docker run --network="host" -p 127.0.0.1:4466:4466/tcp --mount type=bind,source="/home/rene/.minikube",target=$HOME/.minikube --mount type=bind,source="$HOME/.kube",target=/root/.kube kinvolk/headlamp:development /headlamp/server -html-static-dir /headlamp/frontend -plugins-dir=/headlamp/plugins
+```
+
+Then go to https://localhost:4466
+
+
+### Minikube "incluster"
+
+This is if you want to use Headlamp incluster, and test it locally on minikube with a local docker image.
+
+#### docker image in the minikub docker environment
+
+First we have to make the image on the minikube docker environment.
+
+```bash
+$ eval $(minikube docker-env)
+$ DOCKER_IMAGE_VERSION=development make image
+```
+
+#### Create a deployment yaml.
+
+```bash
+$ kubectl create deployment headlamp -n kube-system --image=kinvolk/headlamp:development -o yaml --dry-run -- /headlamp/server -html-static-dir /headlamp/frontend -in-cluster -plugins-dir=/headlamp/plugins > minikube-headlamp.yaml
+```
+
+To use the local docker image we change the imagePullPolicy to Never.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: headlamp
+  name: headlamp
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: headlamp
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: headlamp
+    spec:
+      containers:
+      - command:
+        - /headlamp/server
+        - -html-static-dir
+        - /headlamp/frontend
+        - -in-cluster
+        - -plugins-dir=/headlamp/plugins
+        - -base-url
+        - /headlamp
```

Comment:
This section doesn't seem to be released to the base-url but you're adding this option here.

Reply:
Thanks. I removed this.